content/en/docs/ops/diagnostic-tools/istiod-pod-remote-debug/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Remote Debugging of Istiod Pod
```

Comment:
I suspect this is better suited for https://github.com/istio/istio/wiki

Reply:
Maybe can also link to https://github.com/howardjohn/local-istio-development, since its trivial to debug this way (with the caveat it is not in the remote)

Reply:
Agree this makes sense to be in the wiki (either create or update current pages).

Reply:
@ericvn @howardjohn would need help from someone with push access to the wiki to copy this content there.

Reply:
Let me take a few minutes to see about moving what's here. I made a few minor changes to get this: https://github.com/ericvn/istio/wiki. Just need to verify moving the images over.